Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -145,6 +169,107 @@ - (NSError *)waitForDeviceReady {
     return nil;
 }
 
++ (BOOL)installAppWithBundleID:(NSString *)hostBundleID
```

Comment:
This is a new method added, not merge conflict. but it is OK to change it to a instance method.